index.html

```diff
@@ -75,7 +75,7 @@ <h2>Stages</h2>
     <td>
       <ul>
         <li>Identified a champion or champion group who will advance the addition
-        <li>Prose outlining the problem or need and the general shape of a solution
+        <li>Prose outlining the problem or need, and the general shape of a solution
```

Comment:
Hmm, this comma doesn't seem right. You could replace "or" with a slash if you wanted?

Reply:
seems right to me? a slash would also be fine tho.

Reply:
I read it as unembedding the 'and', i.e. "Prose ... and the general shape" instead of "the problem/need and the general shape".

Reply:
are you assuming an oxford comma is not always present?

Reply:
I don't follow; there aren't three conjuncts.

Reply:
right - i'm having trouble seeing your reading of it. I always read it as "prose outlining $something and the general shape of a solution", and $something is ofc "the problem or need".

Reply:
https://apastyle.apa.org/learn/faqs/use-slash-mark

Reply:
Jordan is correct.  Without the comma, this could be read as:

> Prose outlining the problem or (need and the general shape of a solution)
(incorrect)

or 

> Prose outlining the (problem or need) and the general shape of a solution
(correct)

Reply:
I'm afraid you're mistaken. I understood that your concern must be or/and precedence, hence my presenting an alternative. The issue is that the comma only _introduces_ ambiguity: `Prose outlining the problem or (need and the general shape of a solution)` is an ungrammatical parse (it would need be "the need and general shape"), whereas `Prose outlining the problem or need, and the general shape of a solution` is a natural one-liner version of:

> Entrance Criteria:
> - Prose outlining the problem or need
> - The general shape of a solution

Reply:
I disagree, but do you have a concrete suggestion for an improved version?

Reply:
alright, APA says no, but CMOS says it's ok.  your slash is granted.

Reply:
I was merely saying not to insert the comma; the slash was my olive branch...